src/js/worklets/shell/floating_screen.js

```diff
@@ -106,3 +106,52 @@ export function screenBorderRadius(screenState) {
     }
   });
 }
+
+export function screenGestureOnUpdate(screenLeft) {
+  return function (event) {
```

Comment:
oops forgot to add, 'worklet' directive

Reply:
fixed in https://github.com/status-im/status-mobile/pull/16390/commits/68ced940bdd38b8d894cccc791b77cef79f02c0f